include/pq_flash_index.h

```diff
@@ -94,6 +94,10 @@ template <typename T, typename LabelT = uint32_t> class PQFlashIndex
 
     DISKANN_DLLEXPORT diskann::Metric get_metric();
 
+    DISKANN_DLLEXPORT std::vector<T> get_vector(std::uint64_t VID);
+    DISKANN_DLLEXPORT std::vector<std::uint8_t> get_pq_vector(std::uint64_t VID);
+    DISKANN_DLLEXPORT std::vector<std::uint64_t> get_neighbors(std::uint64_t VID);
```

Comment:
Should get_neighbors return vector of uint32_t? Or are you trying to make this future-proof?